File: app/src/main/java/com/timelysoft/tsjdomcom/service/model/PersonModel.java
package com.timelysoft.tsjdomcom.service.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class PersonModel {
    @SerializedName("Id")
    @Expose
    private Integer id;
    @SerializedName("DateOfBirth")
    @Expose
    private String dateOfBirth;
    @SerializedName("FullName")
    @Expose
    private String fullName;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getDateOfBirth() {
        return dateOfBirth;
    }

    public void setDateOfBirth(String dateOfBirth) {
        this.dateOfBirth = dateOfBirth;
    }

    public String getFullName() {
        return fullName;
    }

    public void setFullName(String fullName) {
        this.fullName = fullName;
    }

}

File: app/src/main/java/com/timelysoft/tsjdomcom/service/model/PersonModel.kt
package com.timelysoft.tsjdomcom.service.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class PersonModel {
    @SerializedName("Id")
    @Expose
    var id: Int? = null
    @SerializedName("DateOfBirth")
    @Expose
    var dateOfBirth: String = ""
    @SerializedName("FullName")
    @Expose
    var fullName: String = ""

}